multiwan.rst

```diff
@@ -74,3 +74,20 @@ The ``General settings`` page allow users to specify the following parameters:
   By setting the number of packets, users can control the intensity of the monitoring.
 - Determining unreachability after how many failed tests:  users can configure the system to determine when a WAN connection should be considered unreachable.
   This is done by specifying a threshold - after how many consecutive failed tests the WAN connection is deemed unreachable.
+ 
+
+Reset configuration
+===================
+
+.. warning::
+
+   This will effectively reset the MultiWAN configuration, with a loss of internet connection if no WAN is configured.
+
+In case users need to delete entirely the MultiWAN configuration (if there's the need to disable it or if user want to start over the configuration) the following list of commands is provided to delete such configuration:
```

Comment:
```suggestion
If your firewall was previously configured with two or more WAN interfaces and after reconfiguration there is only one WAN interface, it is recommended to reset the MultiWAN configuration. This will ensure that your firewall is properly configured and functioning as intended.

Execute the following command to reset the MultiWAN configuration: 
```